WorkflowFunctionBundle/Tests/Functional/Repository/WorkflowFunctionRepositoryTest.php

```diff
@@ -2,6 +2,8 @@
 
 namespace OpenOrchestra\WorkflowFunctionBundle\Tests\Functional\Repository;
 
+use OpenOrchestra\ModelInterface\Repository\Configuration\FinderConfiguration;
+use OpenOrchestra\ModelInterface\Repository\Configuration\PaginateFinderConfiguration;
```

Comment:
I think it should not. If A requires B, A shouldn't be aware that B requires C. If A and B requires C, both should mention it.
If A requires B & C but lets B telling about the requirement, there's a flow in the dependancy: if B requires no more C it shouldn't impact A but in that case it will.
Mentionning that A requires both B & C makes B independant on its own requirements. A will not be broken if B doesn't require C no more.

In our case the packages are closely related, but if C was the orchestra-libs for instance, or jms-serializer-bundle, I'm not sure you'll have the same position :)